### ISSUE TITLE
SSL_read()/SS_write() return SSL_ERROR_WANT_ASYNC

### ISSUE THREAD

**Issue by l-w-p** [intel/QAT_Engine#122]
I wrote an application using QAT and SSL_MODE_ASYNC. During testing, SSL_do_handshake returned SSL_ERROR_WANT_ASYNC, but SSL_read / SSL_write never returned SSL_ERROR_WANT_ASYNC, is this a normal result? How can I test if SS_read / SSL_write returns SSL_ERROR_WANT_ASYNC?

**Comment by paulturx** [comment]
Hi 
Sorry for the delay in responding to this.  In order to answer your questions I will need to reproduce - to that end could you please give me details of OS, QAT driver, QAT engine, etc., or else can I assume that it is essentially the same versions as in your issue #123?

**Comment by l-w-p** [comment]
Yes, it's the same version as inissue [#123](https://github.com/intel/QAT_Engine/issues/123)
